upgrade-tidb-using-tiup.md

```diff
@@ -29,14 +29,16 @@ This document is targeted for the following upgrade paths:
 - Support upgrading the versions of TiDB Binlog, TiCDC, TiFlash, and other components.
 - For detailed compatibility changes of different versions, see the [Release Notes](/releases/release-notes.md) of each version. Modify your cluster configuration according to the "Compatibility Changes" section of the corresponding release notes.
 
-> **Note:**
->
-> Do not execute any DDL request during the upgrade, otherwise an undefined behavior issue might occur.
-
 ## Preparations
 
 This section introduces the preparation works needed before upgrading your TiDB cluster, including upgrading TiUP and the TiUP Cluster component.
 
+> **Warning:**
+>
+> - **DO NOT** upgrade a cluster when a DDL statement is being executed in the cluster (usually for the time-consuming DDL statements such `ADD INDEX` and the column type changes).
+> - Before the upgrade, it is recommended to use the [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) command to check whether the cluster has an ongoing DDL job. If the cluster has a DDL job, to upgrade the cluster, wait until the DDL execution is finished or use the [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) command to cancel the DDL job before you upgrade the cluster.
+> - In addition, during the cluster upgrade, **DO NOT** execute any DDL statement. Otherwise, the issue of undefined behavior occurs.
```

Comment:
```suggestion
> - In addition, during the cluster upgrade, **DO NOT** execute any DDL statement. Otherwise, the issue of undefined behavior might occur.
```